src/TaskLineRenderer.ts

```diff
@@ -72,7 +72,7 @@ export class TaskLineRenderer {
      * configuration and a supplied TextRenderer (typically the Obsidian Markdown renderer, but for testing
      * purposes it can be a simpler one).
      *
-     * The element includes the task and its various components (description, priority, block link etc), the
+     * The element includes the task and its various components (description, priority, block link etc.), the
```

Comment:
Reviewing this comment block, I see that the description of `renderTaskLine()` is out-of-date - this bit:

```
.. using the given renderDetails
configuration and a supplied TextRenderer (typically the Obsidian Markdown renderer, but for testing
purposes it can be a simpler one).
```

Those things are supplied to the constuctor now, not this method.

Reply:
Indeed. I removed that, it is described in the constructor.